mlpp_features/nwp.py

```diff
@@ -13,6 +13,14 @@
 xr.set_options(keep_attrs=True)
 
 
+STA_IDs = {
+    "BAS": 75,
+    "GVE": 58,
+    "LUG": 47,
+    "GUT": 79,
+}
+
```

Comment:
I would use the data4web ids. Since everything item is a station, it would simply be 
```suggestion
STA_IDs = {
    "BAS": 175,
    "GVE": 158,
    "LUG": 147,
    "GUT": 179,
}
```
We only need to ensure the correct ids are passed in the json